ui/StatusQ/src/StatusQ/Controls/StatusNavigationButton.qml

```diff
@@ -0,0 +1,46 @@
+import QtQuick 2.14
+
+import StatusQ.Core.Theme 0.1
+
+Rectangle {
```

Comment:
Please use QQC2.Control for new components/controls, not Rectangle

Reply:
Or AbstractButton even (we have a clicked() signal below, some background, image + mouse area as the `contentItem; etc. All typical Button stuff

Reply:
I agree it should be based on QQC2, but I just moved already existing component to StatusQ: 
![image](https://user-images.githubusercontent.com/33099791/196185591-f07b14e4-660d-451b-a40f-8c3313742fd8.png)

Reply:
@caybro anyway addressed: https://github.com/status-im/status-desktop/pull/7929/commits/a8904af1f001942ffe29d2fb2de66487e88b9430